k8s/overlays/azure-dev/common-application.yml

```diff
@@ -0,0 +1,22 @@
+powsybl-ws:
+  environment: dev_
```

Comment:
Don't break compatibility, right now as we do not intend to deploy a second environment on Azure

Reply:
removed